scripts/iast/leak_functions.py

```diff
@@ -34,14 +48,19 @@ def test_iast_leaks(iterations: int, fail_percent: float, print_every: int):
         print("Test %d iterations" % iterations)
         current_rss = 0
         half_rss = 0
+        enable_iast_propagation()
+        from scripts.iast.mod_leak_functions import test_doit
+
+        # TODO(avara1986): pydantic is in the DENY_LIST
+        #  from pydantic import main
+        #  _pre_checks(main, "index_aspect")
 
-        mod = _iast_patched_module("scripts.iast.mod_leak_functions")
```

Comment:
How is the module patched now? I don't see the call to this or the patch_all()

Reply:
We now use enable_iast_propagation(), which is more "realistic" I mean, it simulates a real scenario, unlike _iast_patched_module, which only patches the current script